### DISCUSSION TITLE
更好的ui透明度调整

### DISCUSSION THREAD

**Discussion by E0x72-21** [Meloong-Git/PCL#5280] [提问] [answered]
可否单独在个性化设置里添加个调整卡片和左侧白色菜单透明度的选项（感觉默认的透明度有点太低了
![image](https://github.com/user-attachments/assets/03155f22-90b4-46eb-b942-40f292d1ad02)

**Comment by MoYuan-CN** [comment, marked as answer]
- https://github.com/Hex-Dragon/PCL2/issues/4921

**Reply by E0x72-21** [reply]
>
> 确实重复了，但怎么和一个已忽略的issue绑在一起了，难道是不打算采纳这个建议了吗？
> 

https://github.com/Hex-Dragon/PCL2/issues/4921#issuecomment-2543328247

**Comment by Joker2184** [comment]
~~去除透明度 支持WebView~~
